Update failover-cluster-instance-premium-file-share-manually-configur… (#73703)
Azure/azure-content: articles/azure-sql/virtual-machines/windows/failover-cluster-instance-premium-file-share-manually-configure.md

- Microsoft Distributed Transaction Coordinator (MSDTC) is not supported on Windows Server 2016 and earlier. 
- Filestream isn't supported for a failover cluster with a premium file share. To use filestream, deploy your cluster by using [Storage Spaces Direct](failover-cluster-instance-storage-spaces-direct-manually-configure.md) or [Azure shared disks](failover-cluster-instance-azure-shared-disks-manually-configure.md) instead.
- Only registering with the SQL IaaS Agent extension in [lightweight management mode](sql-server-iaas-agent-extension-automate-management.md#management-modes) is supported. 

> - Database Snapshots are not supported with Azure Files at this time because of Sparse File limitations.  

Reviewer: ```suggestion
- Database Snapshots are not currently supported with [Azure Files due to sparse files limitations](/rest/api/storageservices/features-not-supported-by-the-azure-file-service).  
```
